server/i18n/en.json

```diff
@@ -7942,11 +7938,11 @@
   },
   {
     "id": "ent.elasticsearch.test_config.license.error",
-    "translation": "Your license does not support Elasticsearch."
+    "translation": "Your Mattermost license doesn't support indexed search for ElasticSearch or OpenSearch."
```

Comment:
According to the community report, even the `ElasticSearch` term was causing confusion, so we may need to go a little deeper with the replacements, to essentially eliminate anything that doesn't expect a translation.